### PR TITLE
Bank withdraw

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
Request is sent to WithdrawServlet it uses persistent class which validates the amount of funds, if its valid it withdraws the currency, if it is not it throws an exception, the message of which is displayed to the user(the html itself also validates the amount).

### CODE REVIEW

File: src/main/java/com/clouway/bank/persistent/User.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.clouway.bank.persistent;
+
+/**
+ * @author Krasimir Raikov(raikov.krasimir@gmail.com)
+ */
+public class User {
+  public final String username;
+  public final String password;
+  public final java.lang.Double balance;
```
Comment (EmilGeorgiev) [inline src/main/java/com/clouway/bank/persistent/User.java]: .

File: src/main/java/com/clouway/bank/persistent/User.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.clouway.bank.persistent;
+
+/**
+ * @author Krasimir Raikov(raikov.krasimir@gmail.com)
+ */
+public class User {
```
Comment (EmilGeorgiev) [inline src/main/java/com/clouway/bank/persistent/User.java]: in core package

File: src/main/java/com/clouway/bank/core/AccountRepository.java [hunk 3]
```diff
@@ -18,6 +18,14 @@
...
   Double deposit(String username, String amount) throws ValidationException;
 
+  /**
+   * withdraw funds
+   *
+   * @param username user identification
+   * @param amount   amount of funds to withdraw
+   * @throws ValidationException
+   */
+  Double withdraw(String username, String amount) throws ValidationException;
```
Comment (EmilGeorgiev) [inline src/main/java/com/clouway/bank/core/AccountRepository.java]: create object Amount and pass it here as parameter. It does not look good with these two parameters

File: src/main/java/com/clouway/bank/persistent/User.java (hunk 1 above)
Comment (KrasimirRaikov) [inline src/main/java/com/clouway/bank/persistent/User.java]: I deleted the class, it was a mistake in the rebasing of branches

File: src/main/java/com/clouway/bank/persistent/User.java (hunk 2 above)
Comment (KrasimirRaikov) [inline src/main/java/com/clouway/bank/persistent/User.java]: there is one, this is the mistaken class

File: src/main/java/com/clouway/bank/http/WithdrawServlet.java [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    writer.write(template.evaluate());
+  }
+
+  /**
+   * Loading file as string
+   *
+   * @param filePath the path to the file
+   * @return the string read from the file
+   */
+  private String getHtml(String filePath) {
```
Comment (EmilGeorgiev) [inline src/main/java/com/clouway/bank/http/WithdrawServlet.java]: if this method is duplicated in all servlets we try to think how to avoid it

Comment (KrasimirRaikov) [inline src/main/java/com/clouway/bank/http/WithdrawServlet.java]: I already have in another branch.

File: src/main/java/com/clouway/bank/core/AccountRepository.java (hunk 3 above)
Comment (KrasimirRaikov) [inline src/main/java/com/clouway/bank/core/AccountRepository.java]: Done.

**Comment by EmilGeorgiev** [comment]
Is this last version of PR? If not please update it

File: src/main/java/com/clouway/bank/core/Amount.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+package com.clouway.bank.core;
+
+/**
+ * @author Krasimir Raikov(raikov.krasimir@gmail.com)
+ */
+public class Amount {
+
+  public final String username;
+  public final String amount;
```
Comment (EmilGeorgiev) [inline src/main/java/com/clouway/bank/core/Amount.java]: value

**Comment by EmilGeorgiev** [comment]
LGTM

File: src/main/java/com/clouway/bank/core/Amount.java (hunk 5 above)
Comment (KrasimirRaikov) [inline src/main/java/com/clouway/bank/core/Amount.java]: Done.

**Pull request merged**